src/cli.ts

```diff
@@ -3,16 +3,22 @@
 import { JSONSchema4 } from 'json-schema'
 import minimist = require('minimist')
 import { readFile, writeFile } from 'mz/fs'
-import { resolve } from 'path'
+import * as _glob from 'glob'
+import { promisify } from 'util'
+import { join, resolve, basename } from 'path'
 import stdin = require('stdin')
 import { compile, Options } from './index'
 import { whiteBright } from 'cli-color'
 
+const glob = promisify(_glob)
+
 main(minimist(process.argv.slice(2), {
   alias: {
     help: ['h'],
     input: ['i'],
-    output: ['o']
+    output: ['o'],
+    inDir: ['x'],
+    outDir: ['y']
```

Comment:
Would it work if we reuse the existing `i` and `o` args, and avoid introducing `x` and `y`?

Reply:
Yes probably a good idea!